docs/REST.md

```diff
@@ -0,0 +1,51 @@
+# REST interfaces
+
+## Repair scheduler
+
+The REST interface for the repair scheduler is located under the path `<host>/repair-scheduler/v1/`.
+The following sub-paths exists:
+* `/get/<keyspace>/<table>`
+* `/list[/<keyspace>]`
+
+The interface is only exposing state for scheduled tables.
+
+
+### Get
+
+When performing GET on `<host>/repair-scheduler/v1/get/mykeyspace/mytable` a JSON object of the RepairJob type will be returned.
```

Comment:
Something in this sentence to denote that RepairJob type is a type later description would be helpful. Is it possible to create a link there?